### PR TITLE
Add test skeleton for test_disk_eviction_with_node_level_soft_anti_affinity_disabled

### PR DESCRIPTION
**Pull request by PhanLe1010**
longhorn/longhorn#3076

### CODE REVIEW

File: manager/integration/tests/test_node.py [hunk 1]
```diff
@@ -2597,3 +2597,16 @@ def test_node_eviction_multiple_volume(client, core_api, csi_pv, pvc, pod_make,
...
     assert expect_md5sum == created_md5sum2
+
+
+@pytest.mark.skip(reason="TODO") # NOQA
+def test_disk_eviction_with_node_level_soft_anti_affinity_disabled(): # NOQA
+    """
+    Steps:
+
+    1. Disable the setting `Replica Node Level Soft Anti-affinity`
+    2. Create a volume. Make sure there is a replica on each worker node.
```
Comment (khushboo-rancher) [inline manager/integration/tests/test_node.py:2608]: Write some data to the volume.

File: manager/integration/tests/test_node.py [hunk 2]
```diff
@@ -2597,3 +2597,16 @@ def test_node_eviction_multiple_volume(client, core_api, csi_pv, pvc, pod_make,
...
+    3. Add a new schedulable disk to node-1.
+    4. Disable the scheduling and enable eviction for the old disk on node-1.
+    5. Verify that the replica on the old disk move to the new disk
```
Comment (khushboo-rancher) [inline manager/integration/tests/test_node.py:2611]: 6. Make replica count as 1, Delete the replicas on other 2 nodes. Verify the data from the volume.

**Review by khushboo-rancher — COMMENTED** [review]
LGTM in general, suggested two more steps to verify the data of the moved replica.

File: manager/integration/tests/test_node.py (hunk 1 above)
Comment (PhanLe1010) [reply manager/integration/tests/test_node.py:2608]: Thank you! Fixed.

File: manager/integration/tests/test_node.py (hunk 2 above)
Comment (PhanLe1010) [reply manager/integration/tests/test_node.py]: Thank you! Fixed.

**Review by innobead — APPROVED** [review]
LGTM

**Pull request merged**